content/en/users/Rucio/Rucio-Admin/_index.md

```diff
@@ -0,0 +1,368 @@
+>>> CLIENT.add_rse('NEW_RSE')
+True
+>>> CLIENT.add_protocol('NEW_RSE', {'hostname': 'test.org', 'scheme': 'gsiftp', 'prefix': '/filepath/rucio/', 'port': 8443, 'impl': 'rucio.rse.protocols.gfalv2.Default', 'domain': {"wan": {"read": 1, "write": 1,
+"third_party_copy": 0, "delete": 1}, "lan": {"read": 1, "write": 1, "third_party_copy": 0, "delete": 1}}})
+True
+```
+
+## Updating RSE Protocols
+
+On occasion, it may be necessary to change or update an RSE protocol.
+Unlike settings (`rucio-admin rse update`) or attributes (`rucio-admin rse set-attribute`),
+there isn't a direct CLI function for changing a protocol.
+It would therefore be necessary to remove (`rucio-admin rse delete-protocol`)
+and then add (`rucio-admin rse add-protocol`) it again using different information.
+Alternatively, the Python client has additional functionality to directly update or swap the priority of RSE protocols.
+For example to update the `impl` without changing anything else (the `data` argument is used to update the protocol,
+with the other settings used to specify the protocol to change):
+
+```python
+$ python
+>>> from rucio.client import Client
+>>> CLIENT = Client()
+>>> CLIENT.update_protocols(rse='NEW_RSE', scheme='gsiftp', data={'impl': rucio.rse.protocols.gfal.Default'}, hostname='test.org', port=8433)
+True
+```
+
+To swap the priority of two protocols for the third party copy operation:
+
+```python
+$ python
+>>> from rucio.client import Client
+>>> CLIENT = Client()
+>>> CLIENT.swap_protocols(rse='NEW_RSE', domain='wan', operation='third_party_copy', scheme_a='gsiftp', scheme_b='root')
+True
+```
+
+It's also worth noting that when an RSE is deleted using `rucio-admin rse delete`,
+the entry remains in the database.
+This "soft" deletion means that attempting to add a new RSE with the same name as a deleted RSE will fail.
+This is due to the RSE not having a unique name/VO combination.
+In practice it is therefore better to update a badly configured RSE rather than attempting to delete and re-add it.
+However, if the latter method is preferred,
+it is possible manually rename the deleted RSE in the database
+(as there are no foreign key constraints on its name, just the ID and VO) so that the old name can be re-used.
+
+## Basic Usage
+
+This section covers some of the basic Rucio functions that can be run once the VO has accounts and RSEs set up.
+As with the setup, there are many options that won't be covered here.
+For more information refer to either the main documentation or the help for the function in question.
+
+### Daemons
+
+Most operations in Rucio (such as transfers, deletions, rule evaluation)
+require one or more of the
+[daemons](https://rucio.readthedocs.io/en/latest/man/daemons.html) to be
+running in order to take effect. For a multi-VO instance, these should be
+running for all VOs already. However, on new VO's joining Rucio
+some updating of the daemons will be neccessary.
+
+<!-- markdown-link-check-disable -->
```

Comment:
To be removed

Reply:
removed now added exception in the linter